products/utils.py

```diff
@@ -0,0 +1,31 @@
+import jwt
+
+from django.http import JsonResponse
+
+from gream.settings import SECRET_KEY, ALGORITHMS
+from users.models   import User
+
+def authorization(func):
+    def wrapper(self, request, *args, **kwargs):
+        token = request.headers.get('Authorization', None)
```

Comment:
`access_token`이라고 명확하게 명시해주세요~

```suggestion
        access_token = request.headers.get('Authorization', None)
```

Reply:
넵 수정하겠습니다!